static/app/components/events/eventTagsAndScreenshot/screenshot.tsx

```diff
@@ -0,0 +1,164 @@
+import {Fragment, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Client} from 'app/api';
+import MenuItemActionLink from 'app/components/actions/menuItemActionLink';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import DropdownLink from 'app/components/dropdownLink';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {Panel, PanelBody, PanelFooter} from 'app/components/panels';
+import {IconDownload, IconEllipsis} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {EventAttachment, Organization, Project} from 'app/types';
+import {Event} from 'app/types/event';
+import withApi from 'app/utils/withApi';
+
+import DataSection from './dataSection';
+import ScreenshotEmpty from './screenshotEmpty';
+import ScreenshotPreview from './screenshotPreview';
+
+type Props = {
+  event: Event;
+  api: Client;
+  orgSlug: Organization['slug'];
+  projectSlug: Project['slug'];
+};
+
+function Screenshot({event, api, orgSlug, projectSlug}: Props) {
+  const [attachments, setAttachments] = useState<EventAttachment[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  async function fetchData() {
+    if (!event) {
+      return;
+    }
+
+    setIsLoading(true);
+
+    try {
+      const response = await api.requestPromise(
+        `/projects/${orgSlug}/${projectSlug}/events/${event.id}/attachments/`
+      );
```

Comment:
Can we do something so that we don't fire this request twice? (both here and in `EventAttachments` section)

Reply:
I was planning to fix it in a follow up PR